src/python_testing/TC_AVSUMTestBase.py

```diff
@@ -0,0 +1,107 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+
+import logging
+
+import chip.clusters as Clusters
+from chip.interaction_model import InteractionModelError, Status
+from mobly import asserts
+
+logger = logging.getLogger(__name__)
+
+
+class AVSUMTestBase:
+
+    async def read_avsum_attribute_expect_success(self, endpoint: int = None, attribute: str = ""):
+        cluster = Clusters.Objects.CameraAvSettingsUserLevelManagement
+        full_attr = getattr(cluster.Attributes, attribute)
```

Comment:
why not just pass this in? Right now, if you get the string wrong, you'll get a weird test failure.

Reply:
Changed throughout to pass the actual attributes.<whatever>